lmdk/modules/up_down_mixer/up_down_mixer_hifi3.c

```diff
@@ -0,0 +1,1915 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
All our module code needs to be in src/audio/ - I think this code is also already upstream in ```src/audio/up_down_mixer/up_down_mixer_hifi3.c``` ?